From 346de6e521601e5b1cdf7f062bc6197596cbc628 Mon Sep 17 00:00:00 2001
From: Abhisit Sangjan <asang@celestica.com>
Date: Mon, 19 Jun 2017 16:39:20 +0700
Subject: [PATCH] hwmon: Update to support lm75b

* Add lm75b
* Disabled detect and addressing for echo to probe
---
 drivers/hwmon/lm75.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/hwmon/lm75.c b/drivers/hwmon/lm75.c
index 479ffbe..c9a12d0 100644
--- a/drivers/hwmon/lm75.c
+++ b/drivers/hwmon/lm75.c
@@ -44,6 +44,7 @@ enum lm75_type {		/* keep sorted in alphabetical order */
 	g751,
 	lm75,
 	lm75a,
+	lm75b,
 	max6625,
 	max6626,
 	mcp980x,
@@ -232,6 +233,10 @@ lm75_probe(struct i2c_client *client, const struct i2c_device_id *id)
 		data->resolution = 9;
 		data->sample_time = HZ / 2;
 		break;
+	case lm75b:
+		data->resolution = 11;
+		data->sample_time = HZ / 4;
+		break;
 	case max6625:
 		data->resolution = 9;
 		data->sample_time = HZ / 4;
@@ -315,6 +320,7 @@ static const struct i2c_device_id lm75_ids[] = {
 	{ "g751", g751, },
 	{ "lm75", lm75, },
 	{ "lm75a", lm75a, },
+	{ "lm75b", lm75b, },
 	{ "max6625", max6625, },
 	{ "max6626", max6626, },
 	{ "mcp980x", mcp980x, },
@@ -465,8 +471,10 @@ static struct i2c_driver lm75_driver = {
 	.probe		= lm75_probe,
 	.remove		= lm75_remove,
 	.id_table	= lm75_ids,
+/*
 	.detect		= lm75_detect,
 	.address_list	= normal_i2c,
+*/
 };
 
 /*-----------------------------------------------------------------------*/
-- 
1.9.1

